data-collection/deploy/module-trusted-advisor.yaml

```
@@ -161,21 +162,28 @@ Resources:
                   payer_id = account["payer_id"]
                   logger.info(f"Collecting data for account: {account_id}")
                   read_ta(account_id, account_name)
-                  upload_to_s3(account_id, payer_id)
+                  upload_to_s3(account_id, payer_id, "data")
+                  # TA Priority Recommendations
+                  recommendations = get_priority_recommendations(account_id)
```

same here. you can use iterator
```suggestion
                  trustedadvisor = assume_role(account_id, "trustedadvisor", REGIONS[0], ROLE_NAME)
                  recommendations = (trustedadvisor
                      .get_paginator('list_recommendations')
                      .paginate(type='priority')
                      .search('recommendationSummaries')
                  )
```